dojo/api_v2/views.py

```diff
@@ -2482,50 +3528,56 @@ def report_generate(request, obj, options):
     test = None
     endpoint = None
     endpoints = None
-    endpoint_all_findings = None
     endpoint_monthly_counts = None
-    endpoint_active_findings = None
-    accepted_findings = None
-    open_findings = None
-    closed_findings = None
-    verified_findings = None
-    report_title = None
-    report_subtitle = None
 
     include_finding_notes = False
     include_finding_images = False
     include_executive_summary = False
     include_table_of_contents = False
 
     report_info = "Generated By %s on %s" % (
-        user.get_full_name(), (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")))
+        user.get_full_name(),
+        (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")),
+    )
 
     # generate = "_generate" in request.GET
     report_name = str(obj)
-    report_type = type(obj).__name__
+    type(obj).__name__
```

Comment:
```suggestion

```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb